react/react1/week1/static-todo/src/Todo.jsx

```diff
@@ -0,0 +1,17 @@
+import TodoTitle from "./TodoTitle";
+import TodoDetails from "./TodoDetails";
+
+export default function Todo({ todos }) {
+  const items = [];
+  todos.forEach((todo) => {
+    items.push(<TodoTitle title={todo.title} key={todo.title} />);
+    items.push(
+      <TodoDetails
+        description={todo.description}
```

Comment:
Hi Anna,
Thanks for your valueable comment. I think you mean on line 10 I can use todo ={todo} instead of description={todo.description} deadline={todo.deadline}. Then I think I also need to edit TodoDetails according to that.

Reply:
Precisely!
I know it might sound crazy because you just changed the component you're accessing the prop, but I have 2 main reasons to ask for this small modification:
1 - You don't really need the item description and deadline on the `Todo.jsx`
2 - In the future, you'll treat much bigger and more complex objects, and passing the props destructured as you were, could make the code messy.
For example:
I have the object anna:

```
const anna = {
  first_name: "Anna",
  last_name: "Sobral",
  profession: "Frontend Engineer",
  age: 26,
  hobbies: ["play Nintendo switch", "hiking", "read a book"],
  diary:[{
  date: 24-02-2023,
  entry: "Today I reviewed @agkabir's code, he's awesome!"
  },{
  date: 25-02-2023,
  entry: "Today I replied to @agkabir's comment in his PR, he's very interested!"
  }]
}
```
this is a very small object, but can you imagine how many props you could make out of it?
so by just using anna={anna} in a component tree, you would pass everything at once.
I hope this was useful and if you need anything, just pm me on slack! :)

Reply:
Also, I forgot to mention, that there's a warning on the console because you didn't use a key for this component. 
Every time you map an array to create a component, you'll need a key to differentiate it from the others.

Reply:
Thank you once again. Thanks for the good explanation. 🙂